README.md

```diff
@@ -11,15 +11,15 @@ See the [API docs](https://docs.rev.ai) for more information about the API.
 The recommended way to use the Rev AI Java SDK is to import it into the project using Maven.
 
       <dependency>
-        <groupId>ai.rev.speechtotext</groupId>
-        <artifactId>revai-java-sdk-speechtotext</artifactId>
-        <version>1.4.0</version>
+        <groupId>ai.rev</groupId>
+        <artifactId>revai-java-sdk</artifactId>
+        <version>2.1.0</version>
       </dependency>
 
 ## Build and install locally from source
 Once you've cloned the repo you can use Maven to build it locally and install it in your local Maven .m2 repository.
 
-    mvn install -Dmaven.test.skip=true
+    mvn install -DskipTests=true      
```

Comment:
Did the previous instruction not work for you?

Reply:
Yeah I've never seen it with that option before. I get `Unknown lifecycle phase ".test.skip=true"` and I'm on a fresh install of maven. Not sure where it came from though, maybe it works with some other version?